src/network/connection.h

```diff
@@ -32,6 +32,95 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <vector>
 #include <map>
 
+#define MAX_UDP_PEERS 65535
```

Comment:
Note: Moved here for cleaner file structure:

1. preprocessor   <----
2. typedefs / enums
3. constants
4. class/struct definitions